platform/frontend_connectors/schema_transformer.go

```diff
@@ -1136,10 +1136,12 @@ func (s *SchemaCheckPass) applyMatchOperator(indexSchema schema.Schema, query *m
 			case schema.QuesmaTypeKeyword.Name:
 				return equal()
 			default:
-				// ILIKE '%%' has terrible performance, but semantically means "is not null", hence this transformation
-				if rhsValue == "%%" {
+				if rhsValue == "%%" { // ILIKE '%%' has terrible performance, but semantically means "is not null", hence this transformation
 					return model.NewInfixExpr(lhs, "IS", model.NewLiteral("NOT NULL"))
 				}
+				// we might investigate the potential performance gain of checking
+				// that if rhsValue doesn't contain '%' we could use '=' instead of 'ILIKE'
+				// *however* that'd require few tweaks in the parser
```

Comment:
hmm, seems that this `iLike` is no longer `iLike`. What would happen if we would return `equal` here?